pushflatpakscript/src/pushflatpakscript/flathub.py

```diff
@@ -138,6 +138,23 @@ def files(tar, path):
     return os.path.join(flatpak_tar_basedir, flatpak_deflated_dir)
 
 
+def check_app_id_matches_flatpak(context, flatpak_path):
+    # Extract all ostree refs from the supplied Flatpak repo
+    flatpak_refs = subprocess.check_output(["ostree", "refs"], cwd=flatpak_path).decode().splitlines()
```

Comment:
Actually, let's install `ostree` explicitly in the Dockerfile, to avoid bad surprises later on.

Reply:
> Actually, let's install `ostree` explicitly in the Dockerfile, to avoid bad surprises later on.

See PR #860.
There's also a PR in the scriptworker repo to go with this.